README.md

```diff
@@ -90,8 +90,9 @@ For more details, see https://wiki.jenkins-ci.org/display/JENKINS/GitHub+pull+re
 * Under Advanced, set ``Name`` to ``origin`` and:
   * If you **just** want to build PRs, set ``refspec`` to ``+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*``
   * If you want to build PRs **and** branches, set ``refspec`` to ``+refs/heads/*:refs/remotes/origin/* +refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*`` (see note below about [parameterized builds](#parameterized-builds))
-* In ``Branch Specifier``, enter ``${sha1}`` instead of the default ``*/master``.
-* If you want to use the actual commit in the pull request, use ``${ghprbActualCommit}`` instead of ``${sha1}``
+* In ``Branch Specifier``, instead of the default ``*/master``, enter
+  * ``${ghprbActualCommit}`` if you want to use the head of the pull request branch (e.g. ``/origin/pr/4/head``); or
```

Comment:
I recommend changing `/origin/pr/4/head` to `refs/pull/4/head` which is how GitHub stores the reference.  To verify this:

- Clone GHPRB from GitHub.
- Execute `git ls-remote origin | grep 'pull'` to see all of the proposed pull requests.

I recommend making the same change for `*/merge` examples as well in this documentation change.